### ISSUE TITLE
Badger.ExperimentEditor: Tighter CNTK integration

### ISSUE THREAD

**Issue by borjafdezgauna** [borjafdezgauna/RLSimion#122]
Whenever possible, parameters in a network should be defined as ConfigNodes (DoubleConfigViewModel, BoolConfigViewModel,...) to be able to fork them

**Comment by borjafdezgauna** [comment]
Not a priority right now

**Comment by borjafdezgauna** [comment]
It has become apparent that this is far more important than I thought

**Comment by borjafdezgauna** [comment]
TODO:
1. Set the learning rate each training step (better change the interface i think)
2. Test most basic algorithms: DQN and DeepCACLA
3. Implement continuous-Q(s,a) specific methods: gradient
4. Implement DDPG specific method: soft update

**Comment by borjafdezgauna** [comment]
TODO:
1. Test most basic algorithms: DQN and DeepCACLA
2. Implement continuous-Q(s,a) specific methods: gradient
3. Implement DDPG specific method: soft update

**Comment by borjafdezgauna** [comment]
TODO:
2. Implement continuous-Q(s,a) specific methods: gradient
3. Implement DDPG specific method: soft update
4. Clean-up obsolete classes

**Comment by borjafdezgauna** [comment]
TODO:
1. Test Deep-CACLA
2. Implement normalization
3. Re-test DQN, Double-DQN and Deep-CACLA using normalization
3. Implement continuous-Q(s,a) specific methods: gradient
4. Implement DDPG specific method: soft update
5. Test DDPG
6. Clean-up obsolete classes

**Comment by borjafdezgauna** [comment]
Soft update not implemented yet but not as important as the rest. I'm creating a new issue and closing this one
